x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/datafeed/DatafeedConfigTests.java

```diff
@@ -228,15 +229,15 @@ public void testPastQueryConfigParse() throws IOException {
 
             DatafeedConfig config = DatafeedConfig.LENIENT_PARSER.apply(parser, null).build();
             ElasticsearchException e = expectThrows(ElasticsearchException.class, () -> config.getParsedQuery());
-            assertEquals("[match] query doesn't support multiple fields, found [query] and [type]", e.getMessage());
+            assertEquals("[match] query doesn't support multiple fields, found [query] and [type]", e.getCause().getMessage());
```

Comment:
It's probably best to assert that `e.getCause()` is not `null` before this line and print `e`, otherwise if it ever does fail there will just be a NPE with no indication of what the unexpected exception was.

Same on lines 252 and 447